config/html-webpack-inline-critical-css-plugin.js

```diff
@@ -0,0 +1,255 @@
+const fs = require('fs');
+const path = require('path');
+const parse5 = require('parse5');
+const nwmatcher = require('nwmatcher');
+const css = require('css');
+
+const treeUtils = parse5.treeAdapters.htmlparser2;
+
+const PLUGIN_NAME = 'html-webpack-inline-critical-css-plugin';
+
+const PARSE5_OPTS = {
+  treeAdapter: treeUtils
+};
+
+function defineProperties(obj, properties) {
+  for (const i in properties) {
+    const value = properties[i];
+    Object.defineProperty(obj, i, typeof value === 'function' ? { value: value } : value);
+  }
+}
+
+const ElementExtensions = {
+  nodeName: {
+    get: function() {
+      return this.tagName;
+    }
+  },
+  appendChild: function (child) {
```

Comment:
I can haz `appendChild(child) { /*...*/ }` syntax?

Reply:
Just looked and its Node 4+ so yes!

Reply:
Out of curiosity — what is your baseline for build tooling like this? My gut feeling would be LTS, so Node 8.

Reply:
Nvm, asked this in chat.